align/config/basic_template.sp

```diff
@@ -1,104 +1,129 @@
 .subckt NMOS_S  D G S
+* @: Generator()
```

Comment:
@kkunal1408 , is this annotation required? If so, let's make it optional.

Reply:
@soneryaldiz Are you talking about generator constraint? I need this information to remove the redundant hierarchies from the library and ask the primitive generator to generate these hierarchies. 
Are you suggesting using folder (if basic_template.sp is in PDK directory) or file-based (all subcircuits in basic_template) approach for automatic annotation of these constraints?

Reply:
Got it. This annotation tells ALIGN that PDK module can generate this primitive procedurally. Is this correct?

Reply:
yes